draft-ietf-rtcweb-jsep.xml

```diff
@@ -3786,7 +3786,8 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             <t>If the m= section is of type audio:
             <list style="symbols">
 
-              <t>For each specified "CN" media format, enable DTX for
+              <t>For each specified "CN" media format, enable Discontinuous
```

Comment:
This seems like it should probably be handled in sec.voiceactivitydetection1, where we first discuss CN.